api/internal/builtins/HelmChartInflationGenerator.go

```diff
@@ -174,7 +177,7 @@ func (p *HelmChartInflationGeneratorPlugin) runHelmCommand(
 			fmt.Errorf(
 				"unable to run: '%s %s' with env=%s (is '%s' installed?): %w",
 				helm, strings.Join(args, " "), env, helm, err),
-			stderr.String(),
+				stderr.String()+stdout.String(),
```

Comment:
I'm curious about the concatenation of the `stdout` content here. What's the scenario this aims to address?

Reply:
`helm --debug` write both to stderr (stack trace) and to stdout for the incomplete YAML template.

Reply:
I see. Perhaps there would be value to adding a prefix so the user can identify which output is `stdout` and which output is `stderr`?
I am a bit concerned that this might become a wall of text difficult to parse through.

Reply:
I added an error format. 

The output looks like that now: 
```
Error:  Helm stack trace:
install.go:218: [debug] Original chart version: ""
install.go:235: [debug] CHART PATH: /Users/isarn/temp/helm-debug

Error: YAML parse error on helm-debug/templates/test.yaml: error converting YAML to JSON: yaml: line 4: block sequence entries are not allowed in this context
helm.go:84: [debug] error converting YAML to JSON: yaml: line 4: block sequence entries are not allowed in this context
YAML parse error on helm-debug/templates/test.yaml
helm.sh/helm/v3/pkg/releaseutil.(*manifestFile).sort
        helm.sh/helm/v3/pkg/releaseutil/manifest_sorter.go:146
helm.sh/helm/v3/pkg/releaseutil.SortManifests
        helm.sh/helm/v3/pkg/releaseutil/manifest_sorter.go:106
helm.sh/helm/v3/pkg/action.(*Configuration).renderResources
        helm.sh/helm/v3/pkg/action/action.go:168
helm.sh/helm/v3/pkg/action.(*Install).RunWithContext
        helm.sh/helm/v3/pkg/action/install.go:304
main.runInstall
        helm.sh/helm/v3/cmd/helm/install.go:310
main.newTemplateCmd.func2
        helm.sh/helm/v3/cmd/helm/template.go:95
github.com/spf13/cobra.(*Command).execute
        github.com/spf13/cobra@v1.8.0/command.go:983
github.com/spf13/cobra.(*Command).ExecuteC
        github.com/spf13/cobra@v1.8.0/command.go:1115
github.com/spf13/cobra.(*Command).Execute
        github.com/spf13/cobra@v1.8.0/command.go:1039
main.main
        helm.sh/helm/v3/cmd/helm/helm.go:83
runtime.main
        runtime/proc.go:271
runtime.goexit
        runtime/asm_amd64.s:1695

Helm template:
---
# Source: helm-debug/templates/test.yaml
---
apiVersion: test.kustomize.io/v1
kind: ValuesMergeTest
metadata:
  name: -

: unable to run: 'helm template helm-debug-test /Users/isarn/temp/helm-debug -f /var/folders/6d/ck0zlsjx3vsd4x9r42y6knvr0000gn/T/kustomize-helm-194917191/helm-debug-kustomize-values.yaml --debug' with env=[HELM_CONFIG_HOME=/var/folders/6d/ck0zlsjx3vsd4x9r42y6knvr0000gn/T/kustomize-helm-194917191/helm HELM_CACHE_HOME=/var/folders/6d/ck0zlsjx3vsd4x9r42y6knvr0000gn/T/kustomize-helm-194917191/helm/.cache HELM_DATA_HOME=/var/folders/6d/ck0zlsjx3vsd4x9r42y6knvr0000gn/T/kustomize-helm-194917191/helm/.data] (is 'helm' installed?): exit status 1
```